opsmngr/agent_api_keys.go

```diff
@@ -23,26 +23,24 @@ type AgentAPIKeysServiceOp struct {
 }
 
 type AgentAPIKey struct {
-	ID            string  `json:"id"`
+	ID            string  `json:"_id"`
 	Key           string  `json:"key"`
 	Desc          string  `json:"desc"`
-	CreatedTime   string  `json:"createdTime"`
-	CreatedUserID *string `json:"createdUserId"`
-	CreatedIPAddr *string `json:"createdIpAddr"`
+	CreatedTime   int64   `json:"createdTime"`
+	CreatedUserID *string `json:"createdUserId,omitempty"`
+	CreatedIPAddr *string `json:"createdIpAddr,omitempty"`
```

Comment:
[nit] for strings you don't need the pointer unless an empty string ("") is expected